index.html

```diff
@@ -38,9 +38,12 @@ <h3>I'm a Software Developer.</h3>
             </ul>
         </section>
         <section id="works">
-            <h2>My Recent Works</h2>
+            <div class="works-container00">
+                <h2>My Recent Works</h2>
+                <hr id="hr-works">
+            </div>
```

Comment:
- [ ] Kindly consider to to use hover button interact both disable and hover  button when the My Recent Works button is hovered on with the mouse the description and dark opacity disappears and a My Recent Works button appears  samething applicable to Get in touch button, kindly fis it